lms/djangoapps/commerce/signals.py

```diff
@@ -55,256 +45,3 @@ def handle_refund_order(sender, course_enrollment=None, **kwargs):
                 course_enrollment.user.id,
                 course_enrollment.course_id,
             )
-
-
-# pylint: disable=unused-argument
-@receiver(REFUND_ENTITLEMENT)
-def handle_refund_entitlement(sender, course_entitlement=None, **kwargs):
-    if not is_commerce_service_configured():
-        return
-
-    if course_entitlement and course_entitlement.is_entitlement_refundable():
-        try:
-            request_user = get_request_user()
-            if request_user and course_entitlement.user == request_user:
-                refund_entitlement(course_entitlement)
-        except Exception as exc:  # pylint: disable=broad-except
-            # don't assume the signal was fired with `send_robust`.
-            # avoid blowing up other signal handlers by gracefully
-            # trapping the Exception and logging an error.
-            log.exception(
-                "Unexpected exception while attempting to initiate refund for user [%s], "
-                "course entitlement [%s] message: [%s]",
-                course_entitlement.user.id,
-                course_entitlement.uuid,
-                str(exc)
-            )
-
-
-def get_request_user():
-    """
-    Helper to get the authenticated user from the current HTTP request (if
-    applicable).
-
-    If the requester of an unenrollment is not the same person as the student
-    being unenrolled, we authenticate to the commerce service as the requester.
-    """
-    request = RequestCache.get_current_request()
-    return getattr(request, 'user', None)
```

Comment:
This still seems to be used in this file by `handle_refund_order`.

https://github.com/edx/edx-platform/blob/dd60d693e791b6dae903c6f765c85936fb524a25/lms/djangoapps/commerce/signals.py#L30

Reply:
This was an accidental delete.  Good catch.